### DISCUSSION TITLE
Support Figures

### DISCUSSION THREAD

**Discussion by broqit** [awcodes/filament-tiptap-editor#442] [Ideas]
First of all, thank you for your work on the Filament Tiptap Editor. It’s a fantastic tool that greatly enhances our workflow.

I would like to request the addition of support for the “figures” extension as outlined in the Tiptap documentation [here](https://tiptap.dev/docs/examples/experiments/figure). This feature would be incredibly beneficial for our projects, allowing us to manage images and captions more effectively within the editor.

Thank you for considering this request!

**Comment by awcodes** [comment]
Thank you for the suggestion, but until that extension is released I can't consider adding it to the plugin.
